src/core/nixl_agent.cpp

```diff
@@ -396,6 +396,22 @@ nixlAgent::createBackend(const nixl_backend_t &type,
     return NIXL_ERR_BACKEND;
 }
 
+nixl_status_t
+nixlAgent::getBackendType(const nixlBackendH *backend, nixl_backend_t &type) const {
+    if (!backend) {
+        NIXL_ERROR_FUNC << "backend handle is not provided";
+        return NIXL_ERR_INVALID_PARAM;
+    }
+
+    try {
```

Comment:
Let's remove the `try`.
It is doesn't exist anywhere else in this file.
And it is never used for `getType()`..